Parkinson/Startup/APHProfileExtender.m

```diff
@@ -33,7 +33,17 @@
  
 #import "APHProfileExtender.h"
 
-static  NSInteger  kDefaultNumberOfExtraSections =  2;
+// ***************************************************************************
+// ***************************************************************************
+// Of Note: This class is adding extra sections as of mPower 1.1 because no special
+// options were still required. In order to add special sections, set the
+// kDefaultNumberOfExtraSections to the # required and adjust the decorateCell
+// and didSelectorRowAtIndexPath delegate methods to reflect the new section(s)
+// desired.
+// ***************************************************************************
+// ***************************************************************************
+
+static  NSInteger  kDefaultNumberOfExtraSections =  0;
```

Comment:
I think its related to the spacing, but I didn't investigate that deeply other than making sure the sections after matched what was there before other than the deprecated medication tracker.